db/migrate/20160728193153_add_index_to_pieces.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToPieces < ActiveRecord::Migration
+  def change
+  end
+  add_index :pieces, :game_id
```

Comment:
If you can fix this 1 thing ^. Then this PR is good to merge (go ahead and merge it yourself) 😄